src/helpers/get-custodian-folder-creation-cache-key.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { CUSTODIAN } from '@ukef/constants';
+import * as crypto from 'crypto';
+
+export const getCustodianFolderCreationCacheKey = (parentFolderId: number, folderName: string): string => {
```

Comment:
It might be clearer if this function were renamed `generateCustodianFolderCreationCacheKey` to make it clear that it is actually creating the cache key rather than just getting it via a request, for instance